### PR TITLE
replace standard marker and favorite marker icons

### PR DESCRIPTION
**Pull request by jancborchardt**
Before & after:
capture du 2016-01-03 20-32-40capture du 2016-01-03 20-32-04
More in line with simple ownCloud style. Please review @v1r0x @Henni

### CODE REVIEW

**Comment by v1r0x** [comment]
Looks nice!
A little enhancement: Do we want to have the marker shadow?

**Comment by jancborchardt** [comment]
I removed it because it was also not visible for the favorite markers. But maybe yes, it’s better to have it to visually separate it from the map. Will adjust.

**Comment by v1r0x** [comment]
Any news, @jancborchardt ?

**Comment by Henni** [comment]
Why does this PR modify a 3rdParty file (i.e. `js/3rdparty/leaflet/leaflet-src.js`)?

**Comment by jancborchardt** [comment]
As soon as I add the shadow back, it won't mess with 3rdparty files anymore I think.

**Comment by v1r0x** [comment]
@jancborchardt any update on this?

**Comment by v1r0x** [comment]
@jancborchardt ping?

**Comment by Henni** [comment]
@v1r0x i believe @jancborchardt is on vacation this week

**Comment by v1r0x** [comment]
Does he even work? :D

**Comment by v1r0x** [comment]
@jancborchardt Any news?

**Comment by v1r0x** [comment]
@jancborchardt Hope you are not mad, but I decided to revert the js changes and adjusted the fav icon and added a new shadow marker.
@jancborchardt @Henni What do you think? Ready to merge?

**Comment by jancborchardt** [comment]
@v1r0x nah, I just need to test. :) The favorite marker looks a tad better too.

The SVG files definitely need to be compressed though – using Scour, with the line from core/img/image-optimization.sh, if you want to play around with that. ;)

**Comment by v1r0x** [comment]
@jancborchardt I used scout with the settings from the shell script and also moved the files to another folder. Thus all marker icons are in one folder and not in different folders. I'll squash the commits if you (and/or @Henni) are ok with the changes :)

File: js/3rdparty/leaflet/leaflet-src.js [hunk 1]
```diff
@@ -3421,10 +3421,10 @@ L.icon = function (options) {
...
 
 	options: {
-		iconSize: [25, 41],
-		iconAnchor: [12, 41],
-		popupAnchor: [1, -34],
-
+		iconSize: [25, 41],
+		iconAnchor: [12, 41],
+		popupAnchor: [1, -34],
+
```
Comment (Henni) [inline js/3rdparty/leaflet/leaflet-src.js]: What happend here? Why is this marked in the diff?

**Comment by Henni** [comment]
The navigation markers are broken. Otherwise looks good:
image

File: js/3rdparty/leaflet/leaflet-src.js (hunk 1 above)
Comment (v1r0x) [inline js/3rdparty/leaflet/leaflet-src.js]: Maybe this is from reverting the changes in this file.

**Comment by v1r0x** [comment]
Unfortunately the routing machine uses the hardcoded (or taken from `leaflet.js`) path to the old `.png` file.

**Comment by v1r0x** [comment]
Got it!

**Comment by jancborchardt** [comment]
Ok, let’s get this in – we can always polish more in future pull requests. :)

**Pull request merged**